i18n-helpers/src/directives.rs

```diff
@@ -0,0 +1,130 @@
+use regex::{escape, Regex};
+
+#[derive(Debug, PartialEq)]
+pub enum Directive {
+    Skip,
+    TranslatorComment(String),
+}
+
+pub struct DirectiveFinder {
+    pattern_re: Regex,
+}
+
+impl Default for DirectiveFinder {
+    fn default() -> Self {
+        Self::new("i18")
+    }
+}
+
+impl DirectiveFinder {
+    pub fn new(prefix: &str) -> Self {
+        let pattern: String = r"<!-{2,}\s*".to_owned()
+            + &escape(prefix)
+            + &r"\s*[:-]"
```

Comment:
What does the `-` in the character class match?

Reply:
I wanted to support either `<!-- i18-comment: ... -->` or `<!-- i18:comment: ... -->`; perhaps this is too much flexibility?   I'm very open to simplify this; what would you like?

Reply:
Ah, I see! I perfer the simplest solution with less flexibility — over the years, I've seen that less flexibility normally leads to systems that are easier to understand and maintain :smile:

Reply:
Sounds good; simplifying so it just uses `:` between the "i18n" and the command portion.